### ISSUE TITLE
Fix all a11y tests (we use dev17 now)

### ISSUE THREAD

**Issue by kacperknapik** [SAP/spartacus#6758]
After we switched to dev17, all of the a11y tests are failing now. Tests need some changes - especilly with the raw data - products' names, prices, etc

**Comment by kacperknapik** [comment]
Might be fixed by @hackergil on the backend side
